ruleset.xml

```diff
@@ -243,6 +243,9 @@
     <priority>3</priority>
   </rule>
   <rule ref="rulesets/java/codesize.xml/ModifiedCyclomaticComplexity">
+    <properties>
+      <property name="reportLevel" value="20" />
```

Comment:
Isn't 20 a bit large ? Actually everything over 10 is considered to be overly complex and should be split up. Instead of increasing the number to a senseless value, what's about switching this rule off completely (just to make it clear that we don't care) ?

> Complexity directly affects maintenance costs is determined by the number of decision points in a method plus one for the method entry. The decision points include ‘if’, ‘while’, ‘for’, and ‘case labels’ calls.
> Generally, numbers ranging from 1-4 denote low complexity, 5-7 denote moderate complexity, 8-10 denote high complexity, and 11+ is very high complexity. Modified complexity treats switch statements as a single decision point.

Reply:
We had a recent meeting and agreed to bump adjust PMD rules to avoid slowing down contributions due to debatable PMD suggestions like this one.

Reply:
I would prefer to disable the rule altogether because 20 is completely off any reasonable value.

BTW, I don't get what its so hard to split a method into two, but that's a different kettle of fish.

Actually I would like to switch off PMD altogether, to avoid fruitless discussions like this, as we can't agree on any baseline anyway and just waste energy which we should save for saving issues later.

Reply:
@chirino can you explain why this rule is better with level 20 then with 10, couldn't we just then remove it if it's of no use?
I would think that we want to care about code quality and that this rule at the level of 10 indicates that the code needs to be refactored into smaller, easier to understand and maintain methods.